readme.md

```diff
@@ -79,42 +79,26 @@ Likely supports following social networks:
 
 If you need several Likely widgets on the page, just create another `div` with the class `likely` and list the social networks in it.
 
-### Usage with webpack
+### Using as commonjs module
 
-First, install webpack’s [`json-loader`](https://github.com/webpack/json-loader) and add it to your `webpack.config.js`:
+Likely supports commonjs modules, so you can use it within webpack or browserify build systems.
 
-```sh
-$ npm install json-loader --save-dev
-```
+First, install likely using npm:
 
-```js
-// webpack.config.js
-module.exports = {
-    module: {
-        loaders: [
-            { test: /\.json/, loader: 'json-loader' }
-        ]
-    }
-}
 ```
-
-Then, install `likely` with npm:
-
-```sh
-$ npm install likely --save
+npm install ilyabirman-likely
 ```
 
-Then, initiate Likely:
+Then, use it as commonjs module somewhere in your program:
```

Comment:
“somewhere in your program” doesn’t add any valuable information, so I suppose we can remove this part.

Also, `commonjs` → `a CommonJS`.